### ISSUE TITLE
"wild type genotype link" on meta-genotype page

### ISSUE THREAD

**Issue by ValWood** [pombase/canto#1680]
I'm not sure what this is for, or why it  is there?
![wild type](https://user-images.githubusercontent.com/7359272/48487759-12141d00-e817-11e8-8d1a-8e026e645fad.jpg)

**Comment by ValWood** [comment]
I presume it's for when you  need to create a  pathogen host interaction but you don't have a specific gene altered in the host.  This isn't very clear.

I think I would have an option

OR, no gene specific alleles, select host organism only 
and the ability to select the organism, maybe from a pull down.

Otherwise, how would this work?  If you had multiple host species how would you know which species "wildtype" referred to?

**Comment by kimrutherford** [comment]
> Otherwise, how would this work? If you had multiple host species how would you know which species
> "wildtype" referred to?

If you have more than one host in the session, that "Wildtype genotype" section is hidden until you select a host organism from the drop down.  Perhaps you only had one host?

I agree that the wording could be clearer, especially when you only have one host in the session.

**Comment by jseager7** [comment]
> I think I would have an option ... no gene specific alleles, select host organism only

I agree this is clearer than the current solution. It would also probably be better if the 'wild-type genotype' selection wasn't displayed as a table, but simply as text next to a radio button, like so:

![no_gene_radio](https://user-images.githubusercontent.com/37659591/48550195-f9b10a80-e8c9-11e8-9e3f-3d07d8058a90.PNG)

**Comment by ValWood** [comment]
I agree, that would be better.

**Comment by jseager7** [comment]
@ValWood I've revisited this issue (after two years!) and I've prepared a new design for the wild type genotype picker on the Metagenotype Management page. The new design shows the picker as a list of radio buttons instead of a table (see image below).

With regards to the purpose of the list not being clear, I could add a help icon next to the 'Wild type genotypes' heading explaining that these options are used when no gene-specific alleles are specified in the publication. I think a help icon would keep the user interface less cluttered than having text permanently visible, but if you'd prefer that option, I'm happy to do that.

![image](https://user-images.githubusercontent.com/37659591/115238075-2c076780-a115-11eb-880e-76d9b66b8c57.png)
